notify-service/src/digest_format.py

```
@@ -0,0 +1,45 @@
+
+
+def retry_backoff(attempt):
+    return attempt * 250
+
+
+def poll_interval():
+    return 250
+
+
+def should_resend(days_since):
+    if days_since > 7 == True:
```

Flake8 (7.3.0)

[error] comparison to True should be 'if cond is True:' or 'if cond:'

(E712)